brayns/engine/Scene.cpp

```diff
@@ -440,6 +440,9 @@ void Scene::_updateEnvironmentMap()
     }
 
     if (_backgroundMaterial->isModified())
+    {
+        _backgroundMaterial->resetModified(); // TODO????
```

Comment:
no, to my knowledge that's wrong. All the materials are reset at some point

Reply:
oops... removed